### DISCUSSION TITLE
Localised leakage vent activation based on smoke detector

### DISCUSSION THREAD

**Discussion by SaiSantosh1824** [firemodels/fds#13409] [General]
Hi,
I want to activate localised leakages (Vent to ambient) based on smoke detector activation.
Within the FDS User guide, there is an example for activating an HVAC Damper based on Time in Section 10.2.2.
Kindly provide some guidance on how to do the same based on smoke detector activation.
Thank you

**Comment by drjfloyd** [comment]
exact same process. only you are not referencing a TIME DEVC but your smoke detector.

**Reply by SaiSantosh1824** [reply]
Ahh okay!
Just checking if I have got it right,

If I have the below leakage vent,
&HVAC ID='LL', TYPE_ID='LEAK', VENT_ID='Lift_leakage', VENT2_ID='AMBIENT', AREA=0.06, LEAK_ENTHALPY=.TRUE./
and a smoke detector,
&DEVC ID='SD', PROP_ID='Cleary Photoelectric P1', XYZ=7.0,-7.2,2.4/
&PROP ID='Cleary Photoelectric P1',
      QUANTITY='CHAMBER OBSCURATION',
      ALPHA_E=1.8,
      BETA_E=-1.0,
      ALPHA_C=1.0,
      BETA_C=-0.8/

Should I want to activate the above leakage based on smoke detector activation,
then,
&HVAC ID='LL', TYPE_ID='LEAK', VENT_ID='Lift_leakage', VENT2_ID='AMBIENT', AREA=0.06, LEAK_ENTHALPY=.TRUE., DEVC_ID='SD'/

When I tried the above, I got an error, ERROR(101): Problem with HVAC line number     2, input line number  394.

Kindly suggest where I might be going wrong with the code.

**Reply by drjfloyd** [reply]
Is the line you gave in the post above the second HVAC input?
If you retyped the line in the post, then double check your input file to make sure you don't have any typos there.
If you copy and pasted, carefully check your input file and make sure you have the correct type of quote marks in the input file.
If all else fails you can move the / on the second input to the midpoint of the line, run, and if you get the same error you know it is in inputs to the left (if not to the right). Keep moving till you isolate which input is causing the problem.
